### PR TITLE
[WIP] feat(Ratings and Reviews): Implement displaying of Ratings in Listing and Profile

### PR DESCRIPTION
**Pull request by sianlasaga**
- Add component for displaying a review
- Add component for displaying rating summary
- Add and modify some reusable components
- Some bugfixes regarding the ratings

WIP: Display of ratings in Profile, add reviews pagination

### CODE REVIEW

**Comment by sianlasaga** [comment]
I added some bugfixes for electron

File: src/models/Listing.ts [hunk 1]
```diff
@@ -25,6 +26,9 @@ class Listing implements ListingInterface {
     id: string
   ): Promise<{ profile: Profile; listing: Listing; imageData: [{ src: string }] }> {
     const djaliListingRequest = await Axios.post(`${config.djaliHost}/djali/listing?hash=${id}`)
+    // const djaliListingRequest = await Axios.post(
```
Comment (24thsaint) [inline src/models/Listing.ts]: Debugging lines forgotten to be removed? :))

File: src/models/Listing.ts [hunk 2]
```diff
@@ -218,6 +222,25 @@ class Listing implements ListingInterface {
     const user = await Profile.retrieve()
     return user.peerID === this.vendorID.peerID
   }
+
+  public async getRatings(): Promise<{ ratingSummary: RatingSummary; ratings: Rating[] }> {
+    const { vendorID, slug } = this
+    const ratingSummaryRequest = await Axios.get(
+      `${config.openBazaarHost}/ob/ratings/${vendorID.peerID}/${slug}`
+    )
+    // const ratingSummaryRequest = await Axios.get(
```
Comment (24thsaint) [inline src/models/Listing.ts]: Also here.

File: src/pages/Profile.tsx [hunk 3]
```diff
@@ -56,14 +57,46 @@ class ProfilePage extends Component<CheckoutProps, ProfilePageState> {
...
+        djali.buyerRatings.map(async buyerRating => {
+          const userData = await Profile.retrieve(buyerRating.sourceId)
+          if (userData) {
+            buyerRating.avatar = userData.getAvatarSrc('small')
+            buyerRating.reviewer = userData.name
+          }
+          return buyerRating
+        })
+      )
+      console.log(ratingsSummary)
```
Comment (24thsaint) [inline src/pages/Profile.tsx]: Remove logging.

**Pull request merged**